spec/models/claim/advocate_supplementary_claim_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Claim::AdvocateSupplementaryClaim, type: :model do
+  it_behaves_like 'a base claim'
+
+  specify { expect(subject.external_user_type).to eq(:advocate) }
+  specify { expect(subject.requires_case_type?).to be_falsey } # TODO: SUPPLEMENTARY_CLAIM_TODO to be removed
+  specify { expect(subject.agfs?).to be_truthy }
+  specify { expect(subject.final?).to be_falsey }
+  specify { expect(subject.interim?).to be_falsey }
+
+  describe '#eligible_advocate_categories' do
+    let(:claim) { build(:advocate_supplementary_claim) }
+    let(:advocate_categories) { instance_double(Array) }
+
+    it 'sends message to FetchEligibleAdvocateCategories' do
+      expect(Claims::FetchEligibleAdvocateCategories).to receive(:for).with(claim).and_return(advocate_categories)
+      expect(claim.eligible_advocate_categories).to eq(advocate_categories)
+    end
+  end
+
+  describe '#eligible_misc_fee_types' do
+    let(:claim) { build(:advocate_supplementary_claim) }
```

Comment:
Minor nitpick... this could be declared once at the top rather than in each describe block

Reply:
👍 that was moved in a later commit